_posts/2019/2019-06-23-using-strftime-in-a-rails-view-is-probably-a-mistake.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Using strftime in a Rails view is probably a mistake"
+description: "I heard you like the % character"
+layout: article
+category: ruby
+image:
+  base: '2019/using-strftime-in-a-rails-view-is-probably-a-mistake'
+  alt: 'Old broken clocks'
+  credit:  Heather Zabriskie
+  source: "https://unsplash.com/photos/yBzrPGLjMQw"
+
+---
+
+The `strftime` method appears in many languages, [all the way back to C](https://en.wikipedia.org/wiki/C_date_and_time_functions#strftime)!
+
+The syntax of the (mostly impenetrable) time formatting arguments haven’t changed that much in years and are comprehensively covered in the [Ruby documentation for strftime](https://ruby-doc.org/core-2.6.3/Time.html#method-i-strftime). They're remarkably consistent across languages!
```

Comment:
I wonder if it's worth keeping stuff to do with other languages all together: ie it appears in many languages _and_ it's remarkably consistent AND the arguments haven't changed much and you can find them here.

I'd suggest starting by saying it appears in many places and it's consistent. And then introducing the Ruby docs.